packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/plugin-ledger-connector-fabric.ts

```diff
@@ -219,6 +221,8 @@ export class PluginLedgerConnectorFabric
   private readonly peerBinary: string;
   private readonly goBinary: string;
   private readonly cliContainerGoPath: string;
+  private readonly sshConfig!: SshConfig;
```

Comment:
@jagpreetsinghsasan Is there anything preventing us from definitively assigning these fields in the constructor so that we don't need the `!` syntax?

Reply:
I declared that this field cannot be null via `!` syntax, then assigned this field in the constuctor from the 2 possible options, either from the usual `sshConfig` or from the base64 encoded version of it and finally controlled the flow using `Checks.truthy(this.sshConfig != null, `${fnTag}:sshConfig`);`
I did it in this way as the other way would be to declare the var without `!` and then do some workaround to address the issue of `variable is used before assigning`
![Screenshot (56)](https://github.com/user-attachments/assets/9752cfa6-6920-4ddd-9ab9-57df2cb06d85)
In short, leveraged the use of `Checks.truthy( )` alongwith `var!` to have precise declaration.
Please let me know if this has some flaws/vulnerability or there exists a better way to do the same.

Reply:
@jagpreetsinghsasan The if conditions can be refactored this way and then the compiler no longer has an issue with us dropping the `!` (which should only be used if there is no way to avoid it due it turning off a safety feature of the compiler):

```typescript
if (this.opts.sshConfig) {
    this.sshConfig = this.opts.sshConfig;
} else if (this.opts.sshConfigB64) {
    const sshConfigBuffer = Buffer.from(this.opts.sshConfigB64, "base64");
    const sshConfigString = sshConfigBuffer.toString("utf-8");
    this.sshConfig = JSON.parse(sshConfigString);
} else {
    throw new Error("Cannot instantiate Fabric connector without SSH config");
}

```

I'm guessing the other class member could be refactored the same way so that it does not need the `!`